lexicons/com/atproto/sync/subscribeAllRepos.json

```diff
@@ -28,19 +28,19 @@
                 "rebase"
               ]
             },
-            "repo": {"type": "string"},
-            "commit": {"type": "string"},
-            "prev": {"type": "string"},
+            "repo": {"type": "string", "format": "did"},
+            "commit": {"type": "string", "format": "cid"},
+            "prev": {"type": "string", "format": "cid"},
```

Comment:
Accurate that some of these will soon become cid-internal-refs?

Reply:
yup yup that's right 👍